settings.gradle

```diff
@@ -101,3 +95,8 @@ include ':ktor-network:ktor-network-tls'
 include ':ktor-network:ktor-network-tls:ktor-network-tls-certificates'
 include ':ktor-bom'
 include ':ktor-test-dispatcher'
+include ':ktor-common'
```

Comment:
`ktor-common` is really confusing name here. I can't even guess what should be there

Reply:
This is for sharing code between server and client, and it's too small to be in a top level module. Do you have a better naming ideas?

Reply:
Let's discuss how are we going to have such modules. I'm not sure that we should put all common between all features in a single common module. Maybe it's better to have ktor-serialization-common

Reply:
btw we have a name conflict here: there are 2 ktor-serialization modules(cause all modules start being plain during publishing)